lib/greeve/character/skills.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "../base_item"
+
+module Greeve
+  module Character
+    # Known skills of a character.
+    #
+    # @see https://eveonline-third-party-documentation.readthedocs.io/en/latest/xmlapi/character/char_skills.html
+    class Skills < Greeve::BaseItem
+      # Unpublished skill
+      UNPUBLISHED = 0
+      # Published skill
+      PUBLISHED = 1
+
+      endpoint "char/Skills"
+
+      attribute :free_skill_points, xpath: "eveapi/result/freeSkillPoints/?[0]", type: :integer
+
+      rowset :skills, xpath: "eveapi/result/rowset[@name='skills']" do
+        attribute :type_id,     xpath: "@typeID",      type: :integer
+        attribute :type_name,   xpath: "@typeName",    type: :string
+        attribute :skillpoints, xpath: "@skillpoints", type: :integer
+        attribute :level,       xpath: "@level",       type: :integer
+        attribute :published,   xpath: "@published",   type: :integer
```

Comment:
Should the type be `boolean` instead? It looks like the documentation might be wrong:
https://eveonline-third-party-documentation.readthedocs.io/en/latest/xmlapi/character/char_skills.html